Assets/Scripts/TouchManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TouchState
+{
+    // 参考：https://techblog.gracetory.co.jp/entry/2018/06/04/000000
+
+
+    /// <summary>
+    /// タッチ操作の参照クラス
+    /// </summary>
+
+    public class TouchManager
+    {
+        public bool touchFlag;      // タッチ有無
+        public Vector2 touchPosition;   // タッチ座標
+        public TouchPhase touchPhase;   // タッチ状態
+
+        /// <summary>
+        /// コンストラクタ
+        /// </summary>
+        /// <param name="flag"></param>
+        /// <param name="position"></param>
+        /// <param name="phase"></param>
+        public TouchManager(bool flag = false, Vector2? position = null, TouchPhase phase = TouchPhase.Began)
```

Comment:
```suggestion
        public TouchManager()
```

引数が必要な場面がぱっと思いつかないので、なくしていい気がします
ちゃんと考えてないのでそうでもないかもしれない